python/ray/tests/test_runtime_context.py

```diff
@@ -65,10 +59,17 @@ def current_job_id(self):
         def current_actor_id(self):
             return ray.get_runtime_context().current_actor_id
 
+    @ray.remote
+    def f():
+        assert ray.get_runtime_context().current_actor_id is None
+        assert ray.get_runtime_context().current_task_id is not None
+        assert ray.get_runtime_context().current_node_id is not None
+        assert ray.get_runtime_context().current_job_id is not None
```

Comment:
should we just remove `current` from all of these? it seems sort of redundant

Reply:
Got it. Will fix.